formidable/forms/__init__.py

```diff
@@ -113,10 +114,34 @@ class MyForm(FormidableForm):
     """
 
     @classmethod
-    def to_formidable(cls, label, description=u''):
-        form = Formidable.objects.create(label=label, description=description)
+    def to_formidable(cls, label=None, description=None, instance=None):
+        if not instance:
+            if not label:
+                raise ValueError("On creation mode, label value is mandatory.")
```

Comment:
Nevermind, `ValueError` is fine here. I just had the feeling a `KeyError` would fit more this use-case, but given a second thought: it's not. We can leave `ValueError`. :wink: